src/library/Cardano/CoinSelection/Fee.hs

```diff
@@ -9,36 +9,32 @@
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeFamilies #-}
+{-# OPTIONS_HADDOCK prune #-}
```

Comment:
You know, I don't think it would be that bad to show internal functions in haddock, under an internal functions heading.

Users are probably interested in the precise details of how it is implemented. Because they might need to decide between using one of these algorithms or making their own.

Reply:
I agree here

Reply:
> You know, I don't think it would be that bad to show internal functions in haddock, under an internal functions heading.

I definitely see your point that users might wish to see more internal details. Currently, they can still import the identifier into their own code, so there's no hard restriction.

The rationale behind doing it this way is explained here:
https://github.com/input-output-hk/cardano-coin-selection/pull/40#issuecomment-611970563

We can also slowly "unhide" parts of the API as we go forward in time.